### ISSUE TITLE
Gradient Descent for Spiking Neural Networks

### ISSUE THREAD

**Issue by TMats** [TMats/survey#240]
https://arxiv.org/abs/1706.04698
- Dongsung Huh, Terrence J. Sejnowski
- Submitted on 14 Jun 2017 (v1), last revised 19 Jun 2017 (this version, v2)
